Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -89,12 +90,12 @@ public bool IsBackupExist
         }
 
         protected ConcurrentDictionary<string, object> mLocalBackupDic;
-     
+
         public bool IsLocalBackupExist
         {
             get
             {
-                if (mLocalBackupDic != null)
+                if (mLocalBackupDic != null && mBackupInProgress == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)